src/main/java/Calculator.java

```diff
@@ -0,0 +1,22 @@
+public class Calculator {
+    public int plus(double x, double y) {
+        if(x > Integer.MAX_VALUE - y) throw new IllegalArgumentException("두 수의 합이 계산 가능한 범위를 벗어났습니다.");
+        return (int)(x + y);
+    }
+
+    public int minus(double x, double y) {
+        if(x < Integer.MIN_VALUE + y) throw new IllegalArgumentException("두 수의 차가 계산 가능한 범위를 벗어났습니다.");
+        return (int)(x - y);
+    }
+
+    public int multiply(double x, double y) {
+        if(x * y > Integer.MAX_VALUE || x / y < Integer.MIN_VALUE) throw new IllegalArgumentException("두 수의 곱이 계산 가능한 범위를 벗어났습니다.");
```

Comment:
이런 방식으로 코드를 치게 되면 가독성이 떨어지는 것 같아요! 
```suggestion
        if (x * y > Integer.MAX_VALUE || x / y < Integer.MIN_VALUE) {
            throw new IllegalArgumentException("두 수의 곱이 계산 가능한 범위를 벗어났습니다.");
        }
```

이런식으로 개행도 중요합니다!
결국 개발자는 협업을 해야 하기 때문에 내가 아닌 다른 사람들이 보기 좋은 코드를 짜는 것도 능력 중 하나입니다 ☺️